test/run_pass/test_process_exit_recrusively.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+var assert = require('assert');
+var code = 0;
+var count = 0;
+
+process.on('exit', function(c) {
```

Comment:
I do believe `mustCall` has a signature of `mustCall(fn, exact)` of which the second argument should be the call count expected.